### PR TITLE
Josh acceptance test docs

### PR DESCRIPTION
**Pull request by Capt-Slow**
Description of how to get started with acceptance tests.

### CODE REVIEW

File: developer/testing/testing.md [hunk 1]
```diff
@@ -20,34 +20,34 @@ Shortcut for running the test suite:
...
+Step 5) In order to run some of the tests, it is required you create a guest user and have admin user credentials. These credentials can be placed in:
 
 ```
-/tests/acceptance-tests/config/user-data.yml
+tests/acceptance-tests/config/user-data.yml
 ```
 
+In acceptance testing, the idea is to simulate a users experience to validate functionality on the front end. Some test scripts simulate real user login and
+
+do stuff functionality. The credentials put in this file are safe and cannot be accessed from the outside world. Just remember not to commit credentials
```
Comment (zenweasel) [inline developer/testing/testing.md]: "login and do stuff functionality." ?

File: developer/testing/testing.md [hunk 2]
```diff
@@ -20,34 +20,34 @@ Shortcut for running the test suite:
...
 
-2) Run `sudo vi /etc/paths`
+Step 3) Install the latest Java SE Development Kit. That can be found <http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html>.
 
-3) Add chromedriver path
-
-Step 3) Install Java. That can be found <http://www.oracle.com/technetwork/java/javase/downloads/index.html>.
-
-Step 4) Run Selenium Server (NOTE: your selenium version might differ):
+Step 4) Run Selenium Server (NOTE: your selenium version might differ, also you must be using a ):
```
Comment (zenweasel) [inline developer/testing/testing.md]: what does that "also you must be using a ):" mean?

File: developer/testing/testing.md [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
 
+Integration testing is implemented using the[`meteor --test` with Mocha](https://guide.meteor.com/testing.html) .
+
 ## Integration Testing
 
 Integration testing is implemented using the [`meteor --test` with Mocha](https://guide.meteor.com/testing.html) .
 
+```bash
+SERVER_TEST_REPORTER="dot" meteor test --full-app --driver-package dispatch:mocha"
+```
```
Comment (zenweasel) [inline developer/testing/testing.md]: should have the `--once` parameter and no quotation at the end

File: developer/testing/testing.md [hunk 4]
```diff
@@ -1,9 +1,15 @@
 # Testing
 
+Integration testing is implemented using the[`meteor --test` with Mocha](https://guide.meteor.com/testing.html) .
```
Comment (aaronjudd) [inline developer/testing/testing.md]: This needs cleanup, looks like a borked merge....

**Comment by aaronjudd** [comment]
👍

**Pull request merged**